src/config/index.ts

```diff
@@ -1,30 +1,32 @@
 import dotenv from "dotenv";
 
-// Set the NODE_ENV to 'development' by default
+interface Config {
+  port: number;
+  api: {
+    prefix: string;
+  };
+  logs: {
+    level: string;
+  };
+}
+
 process.env.NODE_ENV = process.env.NODE_ENV || "development";
 
-const envFound = dotenv.config();
+const envFound = dotenv.config({
+  path: `.env.${process.env.NODE_ENV}`,
+});
 if (envFound.error) {
-  // This error should crash whole process
-
   throw new Error("⚠️  Couldn't find .env file  ⚠️");
```

Comment:
This could also be more descriptive

`throw new Error('⚠️  Couldn't find .env.${process.env.NODE_ENV} file  ⚠️');`

Definitely with backticks, could not type it there

Reply:
I am removing the .env.development file and just going with the common convention .env. I'll push an update soon

Reply:
Sure no wahala!!...

Also don't forget to update the `ReadMe` file